azure-pipelines-extensions.yml

```diff
@@ -13,6 +13,11 @@ jobs:
       - template: build/sdl-tasks.yml
       - template: build/npm-build-extensions-steps.yml
 pr:
+    branches: 
+      include:
+        - '*'
     paths:
-        include:
-            - extensions/*
\ No newline at end of file
+      include:
+        - extensions/*
+      exclude: 
+        - '*'
```

Comment:
add a newline

Reply:
Added newline at end of file. For some reason it doesn't show Files Changed tab, but if you look at raw file you'll see it.